postgresql/adapter_test.go

```diff
@@ -691,6 +688,26 @@ func TestUUIDInsert_Issue370(t *testing.T) {
 	}
 }
 
+func TestTextMode_Issue391(t *testing.T) {
```

Comment:
@c2h5oh this is where I test binary mode and text mode with native and JSONB types.